xunit.runner.wpf/Impl/RemoteTestUtil.cs

```diff
@@ -92,10 +92,23 @@ private static ProcessInfo StartWorkerProcess()
             return new ProcessInfo(pipeName, process);
         }
 
+        private void RecycleProcess()
+        {
+            var process = _processInfo?.Process;
+            if (process != null && !process.HasExited)
+            {
+                process.Kill();
```

Comment:
I think we can just kill the old process and spawn a new one for now. There's all sorts of stuff that we could do for resiliency (see the madness in http://source.roslyn.io/#Microsoft.CodeAnalysis.InteractiveFeatures/Interactive/Core/InteractiveHost.cs,cd6b30f815387325), but I wouldn't worry too much about it unless it becomes a problem.